tests/pageObjects/AddUserPage.js

```diff
@@ -0,0 +1,32 @@
+class AddUserPage{
+    constructor() {
+        this.accountButtonSelector = "//button[@id='account-button']";
+        this.dropdownAdminPanelSelector = "//div[contains(@class,'dropdown')]/a[2]";
+        this.usersSidebarMenuItemSelector = "//span[text()='Users']";
+        this.addUserButtonSelector = "//div[contains(@id,'contentHeaderRow')]//div//a";
+        this.emailFieldSelector = "//input[@id='user_email']";
+        this.passwordFieldSelector = "//input[@id='user_password']";
+        this.passwordConfirmFieldSelector = "//input[@id='user_password_confirmation']";
+        this.submitUserBtnSelector = "//form/div[2]/div/button";
+        this.checkUserEmailSelector = "//a[text()='demo@demo.com']";
```

Comment:
Can we have a better locator value here rather than directly using user email address?